model/city.go

```diff
@@ -0,0 +1,22 @@
+package model
```

Comment:
I think we should remove this package and use simple functions like `city` and `country` like I've suggested above.